change/@ni-nimble-components-acb97d9b-ffbe-4f70-8c61-a42f00459247.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Initial SB 8 upgrade",
+  "packageName": "@ni/nimble-components",
+  "email": "1458528+fredvisser@users.noreply.github.com",
+  "dependentChangeType": "none"
```

Comment:
I see it on the current public storybook so it's not related to this change (and could choose to address it separately) but looks like the default sorting behavior changed and now Table ends up after the Table Columns instead of before:
![image](https://github.com/ni/nimble/assets/1588923/0c517215-b9f9-4882-a40c-26e715328df5)

Last storybook before the dependency update:
![image](https://github.com/ni/nimble/assets/1588923/3a1cd63c-fb83-4f0a-b92e-a8b1ba88666e)

Reply:
`Table` ends up after all the other table stories and `Table Column Configuration` ends up in the middle after `Table Column: Anchor`

Reply:
Maybe we should just move all the `Table Column Configuration` stuff to be in `Table`

Reply:
I tried a bunch of things to get the sort order to change, and cannot get anything to move around. 🤷‍♂️

Reply:
Turns out the parameters (including the storySort object) weren't being registered with Storybook. Fixed.